x-pack/plugins/alerting/server/rules_client/lib/siem_legacy_actions/transform_legacy_actions.ts

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
I know that you have test on the `format_legacy_actions.test.ts` we think  that 's nice to have it per file and function make it easier to know what we can change if we need to.